client/postit/src/components/SignUp.js

```diff
@@ -0,0 +1,94 @@
+import 'bootstrap/dist/css/bootstrap.css';
+import { useState } from "react";
+function SignUp({onRegistration}){
+  const [username,setUsername] = useState('');
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+  const handleUsernameInput = (event) =>{
```

Comment:
These three handle functions can be condensed into one, since they all take in a change event, and then set a certain variable to the event target value. Something like this [resource](https://dev.to/deboragaleano/how-to-handle-multiple-inputs-in-react-55el)

Reply:
Thanks, cute refactor